components/org.wso2.carbon.siddhi.editor.core/src/main/resources/web/commons/js/dialog/open-sample-file-dialog.js

```diff
@@ -261,6 +245,64 @@ define(['require', 'lodash','jquery', 'log', 'backbone', 'file_browser', 'worksp
                         }
                     });
                 };
+
+                 function openSample(payload) {
+                    var fileRelativeLocation = "artifacts" +self.app.getPathSeperator() +payload
+                    var workspaceServiceURL = app.config.services.workspace.endpoint;
+                    var openSampleServiceURL = workspaceServiceURL + "/read/sample";
+                    var browserStorage = app.browserStorage;
+
+                    $.ajax({
+                        url: openSampleServiceURL,
+                        type: "POST",
+                        data: fileRelativeLocation,
+                        contentType: "text/plain; charset=utf-8",
+                        async: false,
+                        success: function (data, textStatus, xhr) {
+                            if (xhr.status == 200) {
+                                var file = new File({
+                                    content: data.content
+                                },{
+                                    storage: browserStorage
+                                });
+                                openSampleConfigModal.modal('hide');
+                                app.commandManager.dispatch("create-new-tab", {tabOptions: {file: file}});
+                            } else {
+                                openFileWizardError.text(data.Error);
+                                openFileWizardError.show();
+                            }
+                        },
+                        error: function (res, errorCode, error) {
+                            var msg = _.isString(error) ? error : res.statusText;
+                            if(isJsonString(res.responseText)){
+                                var resObj = JSON.parse(res.responseText);
+                                if(_.has(resObj, 'Error')){
+                                    msg = _.get(resObj, 'Error');
+                                }
+                            }
+                            openFileWizardError.text(msg);
+                            openFileWizardError.show();
+                        }
+                    });
+                };
+
+                function searchSample(sampleContent,searchText){
+                    var filter = searchText.toUpperCase();
+                    var sampleElements = sampleContent[0].getElementsByTagName("li");
+                    var noResultsElement  = sampleFileOpen.find("div").filter("#noResults");
+                    var unmatchedCount=0;
```

Comment:
@kts-desilva  Can you format this code? Please check other files as well.

Reply:
Sure. Fixed in f228ff475e8dd19e1ef997ba24528c91a61ff51e